src/docs/admin/references/NodeSourceStartupScriptReference.adoc

```diff
@@ -0,0 +1,41 @@
+Most cloud node source infrastructures allows the user to customize the VM startup script. 
+Usually, this script is expected to download and install the Java Runtime Environment, then download and launch ProActive node.jar, if not already provided by the VM image.
+The script can use the following arguments: 
+
+- `%nodeJarUrl%`: the value of the infrastructure parameter `nodeJarURL`, it specifies the full URL path for downloading _node.jar_ file.
+- `%protocol%`: the network protocol (pnp or pamr) to be used by the ProActive Node for incoming communications.
+- `%jythonPath%`: the path of jython library on the AWS instances. Its default value is `/tmp/node/lib/jython-standalone-2.7.0.jar/Lib`. 
+- `%rmHostname%`: the value of the infrastructure parameter `rmHostname`, it specifies the hostname or the public IP address of the Resource Manager.
```

Comment:
Wondering if, instead of talking about public IP, we should not say 'refer' to an IP address of the RM accessible from deployed nodes, to leave to the user the interpretation of this field based of his NS deployment scheme.